webpack/ForemanInventoryUpload/Components/InventorySettings/AdvancedSetting/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { selectSettings } from '../InventorySettingsSelectors';
+import { handleToggle } from './AdvancedSettingActions';
+import SwitcherPF4 from '../../../../common/Switcher/SwitcherPF4';
+import { settingsDict } from './AdvancedSettingsConstants';
+
+const AdvancedSetting = ({ setting }) => {
```

Comment:
Can you rename it for now to something like `InventorySetting`? also the files names

Reply:
As this will be an advanced setting in the modal, and there would be other settings that will be `InventorySetting`, will `AdvancedInventorySetting` work?

Reply:
let's leave it this way than :)